src/app/pages/Privacy.svelte

```diff
@@ -0,0 +1,170 @@
+<script>
+  import Button from "../components/Button.svelte";
+</script>
+
+<style>
+  .privacy {
+    width: 50%;
+    margin: auto;
+    margin-top: 20px;
+  }
+
+  .privacy h1 h2 h3 h4 {
+    font-family: Cabin, sans-serif;
+  }
+
+  a {
+    text-decoration: none;
+  }
+</style>
+
+<main>
+  <div class="privacy">
+    <a href="/"><Button color="blue">Back</Button></a>
```

Comment:
If it:
 1. goes somewhere, it's an `<a>`
 2. performs some action (e.g., submit something) it's a `<button>`

This **goes back to the homepage**. But you still want it to look like a button. No worries! That's what we have utility classes for!

```suggestion
    <nav style="text-align: center">
      <a href="/" class="button button--blue">Back</a>
    </nav>
```

I wrapped it in a `<nav>` because it seems the intent was to have some sort of block element so that the button would appear centered on the page.